src/entity/name.ts

```diff
@@ -9,7 +9,7 @@ export class Name implements FakerEntity {
     return [
       'firstName',
       'lastName',
-      'findName',
+      'fullName',
```

Comment:
I agree that `fullName` is better name but unfortunately `findName` is not just an arbitrary name, it is a method name in Faker. 

![image](https://user-images.githubusercontent.com/1243921/81171146-3ea98f00-8fdf-11ea-9eb8-37d704b64693.png)

Reply:
I see, that makes sense, shall I change it to what you suggested, or just close the pr?

Reply:
if you want to make the changes, I'm welcoming that :D but it means that the changes will not only be applied for `findName` but also the other parts.